src/algebra/monoid_algebra/basic.lean

```diff
@@ -1211,8 +1229,6 @@ lemma lift_nc_smul {R : Type*} [add_zero_class G] [semiring R] (f : k →+* R)
   lift_nc (f : k →+ R) g (c • φ) = f c * lift_nc (f : k →+ R) g φ :=
 @monoid_algebra.lift_nc_smul k (multiplicative G) _ _ _ _ f g c φ
 
-variables {k G}
```

Comment:
What's the reason for this change?

Reply:
The output of `#where` at this point in the file is
```lean
namespace add_monoid_algebra

open finset finsupp mul_opposite
variables {k : Type u₁} {G : Type u₂} {R : Type u_1} [semiring k]



end add_monoid_algebra
```
which I interpreted as `k` and `G` are already implicit.  Hence, I removed the variables declaration.

My impression is that this got left over in some refactor.